### DISCUSSION TITLE
Simple Loign io Support?

### DISCUSSION THREAD

**Discussion by DSDV** [dani-garcia/vaultwarden#2666] [Q&A] [answered]
Hi all, 
i read that Bitwarden now supports SimpleLogin.io:
https://bitwarden.com/blog/add-privacy-and-security-using-email-aliases-with-bitwarden/

i am still using keepass2 and think about migrating. 
Therefor my question is as the title says: does vaultwarden support simplelogin?

with kind regards

PS: can you migrate between bit and vault warden?

**Comment by BlackDex** [comment, marked as answer]
The email aliases are all client side, and are supported since there is nothing server side needed.

Migrations from/to Bitwarden/Vaultwarden is possible, but only via export/import, and i suggest to use json format then, since that holds all the data compared to csv which will miss some.

**Reply by DSDV** [reply]
thanks for the info :D
